modules/coreos-layering-configuring.adoc

```diff
@@ -63,7 +63,7 @@ metadata:
     machineconfiguration.openshift.io/role: worker <1>
   name: os-layer-custom
 spec:
-  osImageURL: quay.io/my-registry/custom-image@sha256:306b606615dcf8f0e5e7d87fee3 <2>
+  osImageURL: quay.io/my-registry/custom-image@sha256... <2>
```

Comment:
this is fine either way because user will need to replace this with custom OSImage url they have built.